include/keepkey/board/layout.h

```diff
@@ -119,8 +119,9 @@ void layout_home_reversed(void);
 void animate(void);
 bool is_animating(void);
 void force_animation_start(void);
-void animating_progress_handler(const char *desc, int permil);
+void animating_progress_handler(const char *otp, const char *desc, int permil);
```

Comment:
It's not an overload, it's a redefinition and I'm not sure I see a problem with that. It gives us more flexibility with fonts. char *otp might be a bad name for the parameter but it allows two different fonts on the same screen. If we create a new function, it will essentially duplicate what is there which it pretty inefficient.

Reply:
fixed